### ISSUE TITLE
Number of PerformanceElementTiming entries for background-image that contains multiple layers

### ISSUE THREAD

**Issue by hawkinsw** [w3c/element-timing#35]
I hope that this is a meaningful question. If not, I apologize!

In CSS3 you can (apparently) [specify](https://drafts.csswg.org/css-backgrounds-3/#layering) multiple background images for a single element that will be used to build a layered background.

Does the second Note in [Section 3.2](https://wicg.github.io/element-timing/#sec-modifications-CSS) apply in this situation, too?

**Comment by npm1** [comment]
It does apply for it! See this test for example:
https://github.com/web-platform-tests/wpt/blob/master/element-timing/multiple-background-images.html

Do you think this needs further clarification in the spec?

**Comment by hawkinsw** [comment]
Awesome!

I am only concerned because in CSS3 the image type is no longer limited to urls and can be other things like fragments and solid colors. See [https://www.w3.org/TR/css3-images/](https://www.w3.org/TR/css3-images/). Because of that, speaking in the note only about urls as the type of multiple values seems misleading or limiting. 

In light of that, do you have any thoughts on the content of the Note?

**Comment by npm1** [comment]
Yep, that seems like a different issue. I agree that we should be more specific about the fact that we only observe background images associated to a URL. I don't think the Note in 3.2 is inaccurate, as it says "we assume that there is one image request for each Element that a background-image property affects and for each URL that the background-image property specifies" which in particular implies that there is no image request when there is no URL, right?

But I do think that in https://wicg.github.io/element-timing/#sec-elements-exposed we need to add context about the fact that only background images with URL will be exposed. Does that make sense?

**Comment by hawkinsw** [comment]
Yes, thanks for agreeing! I am sorry that I was being inarticulate and made
it harder than it needed to be! I'm not sure exactly how we should rectify
the situation. If you're okay with it, I can specify some changes and put
them in a PR and we can see what people think?

Thanks again for working through this with me!
Will

**Comment by npm1** [comment]
No, thank you for the feedback :) PR is welcome.
